prometheus-metrics-exporter-common/src/main/java/io/prometheus/metrics/exporter/common/PrometheusHttpScrapeRequest.java

```diff
@@ -0,0 +1,24 @@
+package io.prometheus.metrics.exporter.common;
+
+import io.prometheus.metrics.model.registry.PrometheusScrapeRequest;
+
+public class PrometheusHttpScrapeRequest implements PrometheusScrapeRequest {
```

Comment:
We could use `PrometheusHttpRequest` directly and remove `PrometheusHttpScrapeRequest` and `PrometheusScrapeRequest`.

Reply:
Ah, ok, I see. What about `interface PrometheusHttpRequest extends PrometheusScrapeRequest`? Then we could remove `PrometheusHttpScrapeRequest`.